erc1155/src/lib.rs

```diff
@@ -108,7 +102,7 @@ impl ERC1155TokenBase for ERC1155Token {
                 .unwrap_or(&false)
     }
 
-    fn safe_transfer_from(&mut self, from: &ActorId, to: &ActorId, id: &u128, amount: u128) {
+    fn safe_transfer_from(&mut self, from: &ActorId, to: &ActorId, id: &TokenId, amount: u128) {
```

Comment:
It is not the realization of `safe_transfer`. It's better to call `transfer` for now